### DISCUSSION TITLE
how to set default language in docker compose install?

### DISCUSSION THREAD

**Discussion by deboradff** [ckan/ckan#8364] [Q&A]
I've installed CKAN 2.10.4 using docker compose and  I need to change language to pt_BR. How can I do it if I can't change ckan.ini

**Comment by mebordone** [comment]
I don´t know if its the best way, but I've created a script in the     "ckan-compose/ckan/docker-entrypoint.d" folder

spanish_default.sh
`
#!/bin/bash
sed -i 's/ckan.locale_default = en/ckan.locale_default = es_AR/' /srv/app/ckan.ini
`


it works for me.

**Comment by folhesgabriel** [comment]
@deboradff @mebordone perhaps an easier option would be explicitly define CKAN__LOCALE_DEFAULT parameter in your .env file

Eg.
CKAN__LOCALE_DEFAULT=pt_BR

Note:
I'm assuming you're using default ckan's docker-compose set up with envvars extesion activated. Every parameter in the .env file that has the prefix CKAN__  overwrites ckan.ini default configuration values
